test/testvm.py

```diff
@@ -643,30 +708,35 @@ def _locate_qemu_kvm(self):
         return 'qemu-kvm'
 
     def _start_qemu(self, maintain=False, tty=False, monitor=None):
-        if (not os.path.exists(self._image_root)
-            or not os.path.exists(self._image_kernel)
-            or not os.path.exists(self._image_initrd)):
-
-            if not os.path.exists(self.run_dir):
-                os.makedirs(self.run_dir, 0750)
+        if not os.path.exists(self.run_dir):
+            os.makedirs(self.run_dir, 0750)
 
-            backing_file = os.path.join(self.test_data, "images", os.path.basename(self._image_root))
-            kernel_file = os.path.join(self.test_data, "images", os.path.basename(self._image_kernel))
-            initrd_file = os.path.join(self.test_data, "images", os.path.basename(self._image_initrd))
-            if not os.path.exists(backing_file):
-                raise Failure("Image not found: %s" % backing_file)
-            if not os.path.exists(kernel_file):
-                raise Failure("Kernel not found: %s" % backing_file)
-            if not os.path.exists(initrd_file):
-                raise Failure("Initrd not found: %s" % backing_file)
-            subprocess.check_call([ "qemu-img", "create", "-q",
-                                    "-f", "qcow2",
-                                    "-o", "backing_file=%s" % backing_file,
-                                    self._image_root ])
-            subprocess.check_call([ "ln", "-sf", kernel_file, self._image_kernel ])
-            subprocess.check_call([ "ln", "-sf", initrd_file, self._image_initrd ])
-
-        if not self._lock_resource(self._image_root, exclusive=maintain):
+        drive_to_start = self._image_root
+        if os.path.exists(self._image_image):
+            drive_to_start = self._image_image
```

Comment:
THis should check $TEST_DATA/images/$image.qcow2 for existence, and if it exists, use it as a backing file for self._image_image.